mobile/lib/providers/image/immich_local_image_provider.dart

```diff
@@ -68,34 +60,16 @@ class ImmichLocalImageProvider extends ImageProvider<ImmichLocalImageProvider> {
     }
 
     if (asset.isImage) {
-      /// Using 2K thumbnail for local iOS image to avoid double swiping issue
-      if (Platform.isIOS) {
-        final largeImageBytes = _useOriginal
-            ? await asset.local?.originBytes
-            : await asset.local
-                ?.thumbnailDataWithSize(const ThumbnailSize(3840, 2160));
-
-        if (largeImageBytes == null) {
-          throw StateError(
-            "Loading thumb for local photo ${asset.fileName} failed",
-          );
-        }
-        final buffer = await ui.ImmutableBuffer.fromUint8List(largeImageBytes);
+      final File? file = await asset.local?.originFile;
+      if (file == null) {
+        throw StateError("Opening file for asset ${asset.fileName} failed");
+      }
+      try {
+        final buffer = await ui.ImmutableBuffer.fromFilePath(file.path);
         final codec = await decode(buffer);
```

Comment:
Just checking, but this is only used in the photo viewer, right? Not the timeline, etc.?

Reply:
Correct